portal-api/src/v1/portalActivity-object-generator/activityConstants.js

```diff
@@ -0,0 +1,12 @@
+const PORTAL_ACTIVITY_LABEL = {
+  MIA_SUBMISSION: 'Manual inclusion application submitted to UKEF',
+  MIN_SUBMISSION: 'Manual inclusion notice submitted to UKEF',
+  AIN_SUBMISSION: 'Automatic inclusion notice submitted to UKEF',
+};
+
+const PORTAL_ACTIVITY_TYPE = {
+  NOTICE: 'NOTICE',
+};
+
+exports.PORTAL_ACTIVITY_LABEL = PORTAL_ACTIVITY_LABEL;
+exports.PORTAL_ACTIVITY_TYPE = PORTAL_ACTIVITY_TYPE;
```

Comment:
Could we put this with other constants/enums?

Reply:
tried this with @IgnatG.  It refused to read from the other constant and always said undefined.  So moved it here